src/color-scale/color-scale.js

```diff
@@ -144,7 +147,11 @@ const Self = class ColorScale extends NudeElement {
 				}
 
 				return colors;
-			}
+			},
+			dependencies: ["colors", "steps", "space", "coords"],
```

Comment:
We should have an `additionalDependencies` property rather than having to redefine all of them. Can you file it?

Reply:
Done!

https://github.com/nudeui/element/issues/28